README.md

```diff
@@ -176,18 +197,18 @@ start the server with a docker:
 docker run -d -p 2700:2700 alphacep/kaldi-en:latest
 ```
 
-Then configure the transcription class with the following properly in `~/jigasi/jigasi-home/sip-communicator.properties`:
+Then configure the transcription class with the following property in `~/jigasi/jigasi-home/sip-communicator.properties`:
```

Comment:
Yeah probably we need to change and this one to reference the config in /etc/jitsi/jigasi.